### ISSUE TITLE
UWPX-Client is Public Archive

### ISSUE THREAD

**Issue by roughnecks** [modernxmpp/easy-xmpp-invitation#73]
https://github.com/UWPX/UWPX-Client

Is it still worth listing it?

**Comment by sjehuda** [comment]
I think, that it is worth to list, so that more people be aware of it and consequetnly someone new would continue to further develop it.

**Comment by poVoq** [comment]
You are free to advocate for continued development of UWXP, but the purpose of this software is to recommend clients to users and at the current point in time UWXP is absolutely not recommended for normal users.

**Comment by poVoq** [comment]
It has been officially abandoned by the developer. Once there is a fork that is actively developed and distributed it can be added again. This has nothing to do with "censoring", but rather respecting the wishes of the developer and also not recommending clients that are clearly a security issue to users.

**Comment by sjehuda** [comment]
Not having OMEMO is not a security issue.

Utilizing centralized data mining platform is indeed a security issue.

Please. Do not subvert the context.

**Comment by mwild1** [comment]
I agree that XMPP is a protocol, not a club. This easy-xmpp-invitation project is a project, not XMPP. All of the easy-xmpp-invitation project maintainers, plus other voices in the community, all agree that recommending a client to users that is unmaintained is a poor experience at best, and a security hazard at worst - nobody is making sure UWPX is up to date and secure. This is nothing to do with OMEMO.

This is an open-source project and you are absolutely free (and encouraged, if you wish) to fork it or simply make your own customizations in your deployment. I'm going to lock this issue because both sides have presented their opinions already, and I don't see any further progress being made.
